chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertificationIdController.java

```diff
@@ -330,7 +330,7 @@ private CertificationIdResults findCertificationByProductIds(
         List<CQMMetDTO> cqmDtos = certificationIdManager.getCqmsMetByCertifiedProductIds(productIdList);
 
         boolean isValid = validator.validate(criteriaDtos, cqmDtos, new ArrayList<Integer>(yearSet));
-        results.setIsValid(isValid);
+        results.setValid(isValid);
```

Comment:
Just confirming - this doesn't change the way the field is represented in the JSON result object right? i.e. not backwards-compatibility breaking?

Reply:
No, this object isn't used anywhere except as the response that gets sent back when doing stuff in the API with the CMS ID validation stuff